Formula/coin3d.rb

```diff
@@ -41,7 +41,6 @@ class Coin3d < Formula
     sha256 cellar: :any,                 monterey:       "22b2ebe3fea27b2c2636bc5e963834a5a4e29b2afb5d2a043f997e7fa5d89454"
     sha256 cellar: :any,                 big_sur:        "e823c1170d7caceff04fee98a5947a1333e1d67dc75cecf412811a8c01255275"
     sha256 cellar: :any,                 catalina:       "7cf7ce170be433841406d71f0feba6f0e69ace689986aaa0ce59453ba5294a26"
-    sha256 cellar: :any_skip_relocation, x86_64_linux:   "632d169dcfe3c1b2b18464a93ad1a7ea6faf636ea2ff00d56edf8c9ae3de1a40"
```

Comment:
Does the indentation have to be updated? In other PRs, we just removed the bottle line, so just went with that.